analytic_secondaxis/analytic_secondaxis.py

```diff
@@ -0,0 +1,291 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+# Copyright (c) 2010 Camptocamp SA (http://www.camptocamp.com)
+# Copyright (c) 2015 Taktik SA (http://www.taktik.be)
+# All Right Reserved
+#
+# Author : Joel Grand-guillaume (Camptocamp)
+# Author : Adil Houmadi (Taktik)
+#
+# WARNING: This program as such is intended to be used by professional
+# programmers who take the whole responsability of assessing all potential
+# consequences resulting from its eventual inadequacies and bugs
+# End users who are looking for a ready-to-use solution with commercial
+# garantees and support are strongly adviced to contract a Free Software
+# Service Company
+#
+# This program is Free Software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+#
+##############################################################################
+
+from openerp import models, fields, api
+import openerp.addons.decimal_precision as dp
+
+
+class ProjectActivityAl(models.Model):
+    _name = "project.activity_al"
+    _description = "Second Analytical Axes"
+
+    @api.model
+    def search(self, args, offset=0, limit=None, order=None, count=False):
+        analytic_account_ids = []
```

Comment:
Maybe it's better to have a display_name and change rec_name to that one than performing this complicated searches.